.github/workflows/testing.yml

```diff
@@ -0,0 +1,42 @@
+name: Testing & Linting
+
+on:
+  pull_request:
+    branches:
+      - main
+      - develop
```

Comment:
What's the workflow here? Will each member clone their own repo and issue a PR from that? As an FYI, if you're working on open source projects this tends to be the workflow - only one person (Tech lead) tends to have write access and everyone else issues pr's, even if they work in the same company.

Reply:
We will create separate branches from `develop` for each user story and implementation of user stories will take place in their respective branches. When a story is close to completion, one of the members working on said story will PR its branch to `develop`, and the PR will be merged once it passes review. At the end of a monthly phase, `develop` will be merged into `main` (the branch where CD to AWS will be set up), signifying a "release".

We plan to briefly document this workflow in the repo's README.md that will be written in the next few days.